articles/dashboard/index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Management Dashboard
```

Comment:
Let's use `Dashboard` instead of `Management Dashboard` ([reference](https://github.com/auth0/docs/blob/master/STYLEGUIDE.md#the-dashboard))

Comment applies to the whole doc

Reply:
You forgot to change this one (replace Management Dashboard with Dashboard)